Cargo.toml

```diff
@@ -115,12 +115,9 @@ path = "parity/main.rs"
 name = "parity"
 
 [profile.dev]
-panic = "abort"
 
 [profile.release]
 debug = false
-lto = false
```

Comment:
I'm afraid with lto enabled build machines can ran out of memory.

Reply:
according to [documentation](https://doc.rust-lang.org/cargo/reference/manifest.html#the-profile-sections) `lto = false` is the default